contributing_to_docs/doc_guidelines.adoc

```diff
@@ -268,6 +271,84 @@ another topic, see the following usage:
 
 http://asciidoctor.org/docs/user-manual/#include-multiple
 
+== Writing concepts
+Concepts should support the things that users want to do and should not include
+task information like commands or numbered steps.
+
+== Writing tasks
+Write tasks in the following format.
+
+Task Title: Use a verb in the task title (for example, Create or Creating).
+
+Include a paragraph explaining why the user must perform this task. This should be 1-2 sentences maximum.
+
+.About This Procedure
+
+Include any gotchas (things that could trip up the user or cause the task to fail).
```

Comment:
After talking to Brice, I don't really know that we gain anything by adding this in tasks. I think it will confuse people. I think having 1-3 sentences about why the user would want to perform the task is sufficient and we can include things like gotchas there. If there procedures where there are a lot of things for the user to consider, I'd suggest linking to a separate section. I'm open to suggestions though.